### PR TITLE
[DC-1245] Update setupResourceScripts readme to point to dev

### PR DESCRIPTION
**Pull request by s-rubenstein**
__Jira ticket__: https://broadworkbench.atlassian.net/browse/dc-1245

## Addresses
With the change to move the UI integration tests to point to dev, we need to update the setupResourceScripts README in order to ensure that it is in alignment.

### CODE REVIEW

File: tools/setupResourceScripts/README.md [hunk 1]
```diff
@@ -21,17 +21,11 @@ The script outputs are written to a JSON file in the format:
 ```
 
 ### Set up data repo UI test datasets
-* Run the script against both integration-4 and integration-5 to set up the datasets for the UI tests. You'll need to run it for
-both the azure and gcp datasets. So, a total of 4 runs.
-* Authenticate as the test user that created the azure managed application. Password for account can be found
-on our slack channel.
+* Run the script against dev to set up the datasets for the UI tests. You'll need to run it for both the azure and gcp datasets. So, a total of 4 runs.
```
Comment (snf2ye) [inline tools/setupResourceScripts/README.md]:
```suggestion
* Run the script against dev to set up the datasets for the UI tests. You'll need to run it for both the azure and gcp datasets. So, a total of 2 runs.
```
Or we can just remove that last sentence.

**Review by snf2ye — APPROVED** [review]
I have one requested change, but otherwise LTGM. Thanks!

**Review by fboulnois — APPROVED** [review]
👍

**Review by rushtong — APPROVED** [review]
👍🏽

**Pull request merged**